plugins/Feedback/Feedback.php

```diff
@@ -123,15 +123,19 @@ public function showQuestionBanner()
         $nextReminderDate = $feedbackReminder->getUserOption();
         $now = Date::now()->getTimestamp();
 
-        //user answered question
-        if ($nextReminderDate === self::NEVER_REMIND_ME_AGAIN) {
-            return false;
-        }
-
-        // if is new user or old user field not exist
-        if ($nextReminderDate === false) {
-            //new user extend to 6 month, don't show banner
-            $nextReminder = Date::now()->getStartOfDay()->addMonth(6)->toString('Y-m-d');
+        // If there isn't any reminder date set, or never remind me was selected previously (-1) we determine a new date
+        if ($nextReminderDate === false || $nextReminderDate <= 0) {
+
+            // if user was created within the last 6 months, we set the date to 6 months after his creation date
+            $userCreatedDate = Piwik::getCurrentUserCreationData();
```

Comment:
There's a typo in the method name, it should be `date` probably ? Be good to change it in this release before it ends up being used somewhere else

Reply:
fixed that